addon/services/ember-oauth2.js

```diff
@@ -107,7 +110,7 @@ export default Service.extend(Evented, {
    * @param {Function} callback Optional callback
    */
 
-  handleRedirect: on('redirect', function(hash, callback) {
+  handleRedirect: on('redirect', function (hash, callback) {
```

Comment:
nit: can you remove the spaces between function and the (). That seems like odd formatting to me. I see this in a few places in the changes

Reply:
>  I didn't do this myself. This is just the prettier config that comes out of the box with ember cli.

In a way, this is the default style of ember apps.

We can change prettier config, though.